web/rest/client/src/Controller/Provider/PostHolidaysMassImportAction.php

```diff
@@ -42,27 +45,37 @@ public function __invoke(Request $request)
         }
 
         $calendarId = $request->request->get('calendar');
+        $companyId = $company->getId() ?? -1;
```

Comment:
I don't see the point on continuing without a valid companyId, check in the above condition (line 43) if company has value and it is valid

        $company = $user->getCompany();
        if (!$company || !$company->getId()) {
            throw new NotFoundHttpException('Company not found');
        }

or 

        $company = $user->getCompany();
        if (!$company) {
            throw new NotFoundHttpException('Company not found');
        }

        $companyId = $company->getId();
        if (!$companyId) {
            throw new NotFoundHttpException('Company not valid');
        }

Reply:
Execution is not reaching there if there is no valid company (lines 42-44).

It's a little trick to keep phpstan and psalm relaxed without error supresion. There's no chance that any entity's getId() returns null except if it's new. 

Other options are not that effective:
````
if (!$company || is_null($company->getId)) {
      throw new NotFoundHttpException('Company not found');
}
```
This don't prevents psalm complaining about a possible null.